tools/delete_candidate.php

```diff
@@ -18,11 +18,15 @@
  * @license  Loris license
  * @link     https://www.github.com/aces/Loris-Trunk/
  */
+
+use LORIS\StudyEntities\Candidate\CandID;
```

Comment:
I'm pretty sure this needs to go below the autoload require.

Reply:
@driusan For my personal knowledge, can you explain a bit why? I will take that into account fot future changes.

Reply:
I might be mistaken, it might not try autoloading until the class is referenced with "new", but how would the class be loaded if the autoloader hasn't been setup yet?